conf/nomad/postgres.hcl

```diff
@@ -49,6 +49,14 @@ job "${service_name}" {
       connect {
         sidecar_service {}
       }
+
+      sidecar_task {
+        driver = "docker"
```

Comment:
`driver= "docker"` is the default value, you can remove it. Could you also extract resources to the input variables?
```suggestion
```

Reply:
Yes, I will remove that.